crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -233,6 +236,10 @@ void LateUpdate()
                 Debug.LogError("_viewpoint is null, ocean update will fail.", this);
             }
 
+            LightProbes.GetInterpolatedProbe(_viewpoint.position, null, out _sphericalHarmonicsL2);
```

Comment:
this could be a quite expensive call, seems like its doing a lookup into tetrahedra. should we put it in a profile tag so that people will report it back to us if they see it?

the second arg seems to be there to help speed this up. i wonder if we can give it a renderer? one idea might be to give it one of the ocean tiles' renderer, or maybe theres another we could use?

Reply:
If I remember correctly I couldn't find much documentation on how this works, so I'm happy to do anything that might make this more efficient. :)

Reply:
Cool, nice one. Let me know if i can help

Reply:
I ended up taking this on. I added some comments to the last few commits, but exec summary is i moved the lookup into the postprocess and added profile tags to keep an eye on perf.